fs/src/commonMain/kotlin/com/suparnatural/core/fs/FileSystem.kt

```diff
@@ -34,17 +34,17 @@ expect object FileSystem {
     /**
      * Returns a list of stats for the contents of directory at `path`.
      */
-    fun readDir(path: String): List<StatResult>?
+    fun readDir(directory: String): List<StatResult>?
```

Comment:
I feel that `readDir` at `path` is more clear than `readDir` from `directory` since a path is a path, doesn't matter if it is pointing to a directory or a file.

Reply:
agree. i had to rename it as the js `actual` uses the nodejs [path](https://nodejs.org/api/path.html) module so the names were conflicting. i changed it back & instead renamed the nodejs path import